src/actions/posts.js

```diff
@@ -508,7 +508,7 @@ export function getPostThread(postId, skipAddToChannel = true) {
             {
                 type: PostTypes.GET_POST_THREAD_SUCCESS
             }
-        ]), getState);
+        ], 'BATCH_GET_POST_THREAD'), getState);
```

Comment:
I don't know exactly why this change. Is simply for be more explicit about the action type generated, for redux-devtools and for code clarity, or is there any place where we use this information?

Reply:
This is mainly to use while debugging as is easier to find the action with all the `BATCH` actions everywhere, it does not have any implications

Reply:
Sounds good :)

Reply:
@enahum Thanks for the info. I also learned from this :)